dirty.html

```diff
@@ -195,11 +194,11 @@
  
         </div>
       
-      </div>
-      <div></div>
+      </div> <!--change below to unordered list-->
+      
     </div>
     <div id="footer">
       <div>
-        <div id="span">Web Development Immersive Course Webpage</div> General Assembly London <div id="span">2013</div id="span">
+        <div id="span">Web Development Immersive Course Webpage</div> General Assembly London <div>2013</div><!--removed "id="span""-->
```

Comment:
`id=span` seems to me like a terrible naming, so hard to know why it's there... :(